packages/manager/modules/pci/src/projects/project/storages/databases/database/connectors/edit-connector/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,6 @@
+{
+  "pci_databases_connectors_edit_back": "Connectors",
```

Comment:
**question:** Why its located into a translation file? Looks like a static content/constant.

Reply:
Moved to static here: https://github.com/ovh/manager/pull/6579/commits/deefd28e570922645a1df5eb36318b0d30427fb2